python/dolfin/function/expression.py

```diff
@@ -50,32 +78,38 @@ def __init__(self,
                  shape: tuple = ()):
         """Initialise Expression
 
-        Initialises Expression from address of C function and value shape.
+        Initialises Expression from address of a C function and value shape.
+
+        The majority of users should use this class in conjunction with the
+        ``function.expression.numba_eval`` decorator that creates Numba
+        JIT-compiled evaluation functions.
 
         Parameters
         ---------
         eval_func:
-            Address of compiled C function.
-            C function must accept the following sets of arguments:
-            1. (values_p, x_p, cells_p, num_points, value_size, gdim, num_cells)
-                `values_p` is a pointer to a row major 2D
-                C-style array of `PetscScalar`. The array has shape=(number of points, value size)
-                and has to be filled with custom values in the function body,
-                `x_p` is a pointer to a row major C-style 2D
-                array of `double`. The array has shape=(number of points, geometrical dimension)
-                and represents array of points in physical space at which the Expression
-                is being evaluated,
-                `cells_p` is a pointer to a 1D C-style array of `int`. It is an array
-                of indices of cells where points are evaluated. Value -1 represents
-                cell-independent eval function,
-                `num_points`,
-                `value_size`,
-                `gdim` geometrical dimension of point where expression is evaluated,
-                `num_cells`
+            Address of a compiled function.
+            The function must accept the following arguments:
+            ``(values_p, x_p, cells_p, num_points, value_size, gdim, num_cells)``
+            1. ``values_p`` is a pointer to a row-major array of
+               ``PetscScalar`` of shape ``(num_points, value_size)``.
+               The function itself is responsible for filling ``values_p``
+               with the desired Expression evaluations. ``values_p`` is not
+               zeroed before being passed to the function.
+            2. ``x_p`` is a pointer to a row-major array of ``double`` of shape
+               ``(num_points, gdim)``. The array contains the coordinates
+               of the points at which the expression function should be evaluated.
+            3. ``cells_p`` is a pointer to an array of ``int`` of shape
```

Comment:
What is the difference between double reverse ticks and single reverse ticks?

Reply:
Good question. We are following the numpy style?

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Yes. The difference between one and two ticks is not very clear.

Reply:
It's rst - http://docutils.sourceforge.net/docs/user/rst/quickstart.html#text-styles.

Double back ticks for a fixed-width font.